x/concentrated-liquidity/incentives_test.go

```diff
@@ -2910,6 +2912,7 @@ func (s *KeeperTestSuite) TestCreateIncentive() {
 		recordToSet        types.IncentiveRecord
 		existingRecords    []types.IncentiveRecord
 		minimumGasConsumed uint64
+		authorizedUptimes  []time.Duration
```

Comment:
Note to reviewer: we add this field to test a range of supported uptimes and, importantly, to ensure that existing test cases do not pass or fail trivially due to only authorizing 1 nanosecond uptime after disabling charging